src/js/libcs/OpDrawing.js

```diff
@@ -377,7 +377,9 @@ evaluator.drawconic$1 = function(args, modifs) {
             Conic.matrix = arr;
         }
 
+
     }
+    Conic.matrix = List.normalizeMax(Conic.matrix);
```

Comment:
For a moment I was afraid that this, too, would modify an existing element. But that's just a local object. Which made me think: why do we have that object? Why not simply pass the matrix directly to `eval_helper.drawconic`, without enclosing it into such an object? Not really related to the issue at hand here, just got me wondering.